### ISSUE TITLE
Entries cannot be deleted or moved to folders; adding to folders is unintuitive

### ISSUE THREAD

**Issue by nielsk** [mssun/passforios#58]
There is no way to delete an entry or move it to another folder. When adding an entry you have to call it something like "private/Test" so that the entry gets moved to the private-folder. And this happens only after sync. 
Imho this is fine behavior on the cli but in an iOS-app that should be handled differently by letting the user choose folders and even let them create them new ones.

**Comment by mssun** [comment]
Yes, we definitely plan to have a full functional app. But we are short of hands, some functions are not completed.

**Comment by mssun** [comment]
Support deleting a password: c21502a10f931bbe096a0bfbde8d684413f3c93f

**Comment by nielsk** [comment]
Deletion works now. Didn't check yet completely on a computer though if it is really gone.

**Comment by mssun** [comment]
Support folder operations  in commit: d8ecd1e889bf5b3b268403d65254b13f0ff83b9e

Please try, thanks.

**Comment by mssun** [comment]
Folder operation functions are merged into master branch. A new version is also released via TestFlight. I'm closing this issue. Feel free to open a new if you meet related issues. Thanks.

- You can create an entry. E.g., `dir/password`.
- You can change folder name. E.g., `dir1/password` to `dir2/password`
- You can also delete folders. E.g., `dir1/dir2/password` to `dir1/password`
